src/main/scala/com/fortysevendeg/android/sqlite/SQLDroidConnection.scala

```diff
@@ -0,0 +1,220 @@
+package com.fortysevendeg.android.sqlite
+
+import java.sql._
+import java.util.Properties
+import java.util.concurrent.Executor
+
+import com.fortysevendeg.android.sqlite.logging.{AndroidLogWrapper, LogWrapper}
+import com.fortysevendeg.android.sqlite.metadata.SQLDroidDatabaseMetaData
+import org.sqldroid.{SQLDroidPreparedStatement, SQLDroidStatement}
+
+class SQLDroidConnection(
+  databaseName: String,
+  timeout: Long = 0,
+  retryInterval: Int = 50,
+  flags: Int = 0,
+  val logWrapper: LogWrapper = new AndroidLogWrapper()
+  ) extends Connection with WrapperNotSupported {
+
+  // TODO - Method for compatibility with java classes. Remove after migration of this classes
```

Comment:
Not yet, the dependency is with the the last two migrations (`Statement` and `PreparedStatement`)